src/mqtt.c

```diff
@@ -494,6 +523,10 @@ static int mqtt_write (const data_set_t *ds, const value_list_t *vl,
  *   StoreRates true
  *   Retain false
  *   QoS 0
+ *   CAfile "ca.pem"			Enables TLS if set
```

Comment:
Please add these to `src/collectd.conf.pod`, too. Please also consider to default on [Apache's names for these settings](http://httpd.apache.org/docs/2.2/mod/mod_ssl.html) – setting up TLS is, apparently, hard enough, so I'd like to make it as easy for the users as possible.